integration-tests/tests.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/python3
+
+import subprocess, argparse, shlex, sys, re
+
+def run_tests(behaviour, args):
+    if behaviour in {'put', 'get'}:
+        try:
+            print(f'Running bobp -b {str(behaviour)} {args.rstrip()}')
+            p = subprocess.check_output(shlex.split(f'./bobp -b {behaviour} {args.rstrip()}'))
+            print(str(p))
+            if f'{behaviour} errors:' in str(p) or f'panicked' in str(p):
+                sys.exit(f'{behaviour} test failed, see output')
+        except subprocess.CalledProcessError as e:
+            sys.exit(str(e.stderr))
+    elif behaviour == 'exist':
+        try:
+            print(f'Running bobp -b {str(behaviour)} {args.rstrip()}')
+            p = subprocess.check_output(shlex.split(f'./bobp -b {behaviour} {args.rstrip()}'))
+            print(str(p))
+            found_exist = re.search(r'\b[0-9]{1,}\sof\s[0-9]{1,}\b', str(p))
+            if not found_exist:
+                sys.exit(f"No {behaviour} output captured, check output")
+            exists = found_exist.group(0).split(' of ')
+            if exists[0] != exists[1]:
+                sys.exit(f"{exists[0]} of {exists[1]} keys, {behaviour} test failed, see output")
+            else:
+                print(f"{exists[0]} of {exists[1]} keys")
+        except subprocess.CalledProcessError as e:
+            sys.exit(str(e.stderr))
+        except Exception as e:
+            sys.exit(str(e))
+
+def get_run_args(mode, args):
+    return {'-c':args.count, '-l':args.payload, '-h':f'{args.node}', '-s':args.start, '-e':args.end, '-t':args.threads, '--mode':args.mode, '-k':args.keysize, '-p':test_run_config.get(mode)}
+
+test_run_config = {'put':'20000', 'get':'20001'}
+
+parser = argparse.ArgumentParser(description='This script launches bob tests with given configuration.')
+parser.add_argument('-c', dest='count', type=int, help='amount of entries to process', required=True)
+parser.add_argument('-l', dest='payload', type=int, help='payload in bytes', required=True)
+parser.add_argument('-n', dest='node', type=str, help='target node address', required=True)
```

Comment:
I suggest to pass nodes list and choose different nodes for `put`, `get` and `exist` tests